ship.config.js

```diff
@@ -1,35 +1,49 @@
 /* eslint-disable import/no-commonjs */
 
-const fs = require('fs');
-const path = require('path');
+const packages = [
+  'packages/autocomplete-core',
+  'packages/autocomplete-preset-algolia',
+  'packages/autocomplete-react',
+];
 
 module.exports = {
+  monorepo: {
+    mainVersionFile: 'lerna.json',
+    packagesToBump: packages,
+    packagesToPublish: packages,
+  },
   mergeStrategy: {
     toSameBranch: ['next'],
   },
   publishCommand({ tag }) {
     return `yarn publish --access public --tag ${tag}`;
   },
-  versionUpdated({ version, dir }) {
-    // Bump the string version in the version file
-    const versionPath = path.resolve(dir, 'src/version.ts');
-    fs.writeFileSync(versionPath, `export const version = '${version}';\n`);
-  },
-  // @TODO: this will be possible once this is addressed:
-  // https://github.com/algolia/shipjs/issues/604
-  // afterPublish({ exec, dir }) {
-  //   // Update the Autocomplete.js version in the examples
-  //   const examplePath = path.resolve(dir, 'examples/autocomplete.js');
-  //   const { version } = require('./package.json');
+  versionUpdated({ version, exec }) {
+    const update = ({ package, deps }) => {
+      /*
+        With caret(^), the dependencies need to be quoted, like the following:
+        > yarn workspace abc add "def@^1.0.0" "ghi@^1.0.0"
+      */
+      const depsWithVersion = deps.map(dep => `"${dep}@^${version}"`).join(' ');
+      exec(`yarn workspace ${package} add ${depsWithVersion}`);
+    };
 
-  //   // eslint-disable-next-line no-console
-  //   console.log('Updating Autocomplete.js dependency in examples...');
+    exec('eslint lerna.json --fix');
```

Comment:
What is this line for? Do you mind adding a comment?

Reply:
Here it is https://github.com/francoischalifour/autocomplete.js/pull/37/commits/b914d439f20a9ad15f1fbd427b73d23b892f2e70